### DISCUSSION TITLE
Not received your education benefits

### DISCUSSION THREAD

**Discussion by kshitij-vk** [community/community#148152] [GitHub Education]
### Hi everyone,

I hope this message finds you well. I submitted my application for the GitHub Education benefits and understand it typically takes about three days for processing. However, I have not yet received a confirmation or update regarding my application.

I would appreciate it if you could kindly check on the status of my application or let me know if any additional information is required from my side to complete the process.

Thank you for your time and assistance!

**Comment by httperror505** [comment]
Hi @kshitij-vk ,

It may take 5 to 10 business days for verification to be completed, if no update was given you may try to apply again and check if all requirements are met.
